src/sentry/api/fields/avatar.py

```diff
@@ -62,3 +68,35 @@ def is_valid_size(self, width, height):
         if width > self.max_dimension:
             return False
         return True
+
+
+class SentryAppLogoField(AvatarField):
+    def __init__(
+        self,
+        max_size=settings.SENTRY_MAX_AVATAR_SIZE,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.max_size = max_size
+
+    def to_internal_value(self, data):
+        if not data:
+            return None
+        data = b64decode(data)
+        if len(data) > self.max_size:
+            raise ImageTooLarge()
+
+        if SVG_RE.match(data.decode("utf-8")) is not None:
```

Comment:
nit: reverse the condition and you can early return

```suggestion
        if SVG_RE.match(data.decode("utf-8")) is None:
       		return None
```